eurocodedesign/units/__init__.py

```diff
@@ -307,21 +325,21 @@ class Kelvin(AbstractUnit):
 
 
 centimeter = partial(Meter, prefix=Prefix.centi)
-cm: TypeAlias = centimeter
+cm = centimeter
 millimeter = partial(Meter, prefix=Prefix.milli)
-mm: TypeAlias = millimeter
+mm = millimeter
 
 square_centimeter = partial(Meter_2, prefix=Prefix.centi)
-cm2: TypeAlias = square_centimeter
+cm2 = square_centimeter
 square_millimeter = partial(Meter_2, prefix=Prefix.milli)
-mm2: TypeAlias = square_millimeter
+mm2 = square_millimeter
 cubic_centimeter = partial(Meter_3, prefix=Prefix.centi)
-cm3: TypeAlias = cubic_centimeter
+cm3 = cubic_centimeter
 
 
 kiloNewton = partial(Newton, prefix=Prefix.kilo)
-kN: TypeAlias = kiloNewton
+kN = kiloNewton
 
 GigaPascal = partial(Pascal, prefix=Prefix.giga)
```

Comment:
would add `GPa = GigaPascal`

Reply:
I wil do it, but in general it should be done in a seperate commit/issue to add functionality.